### DISCUSSION TITLE
Will an upgrade reset job progress?

### DISCUSSION THREAD

**Discussion by Qhilm** [immich-app/immich#10650] [Q&A] [answered]
I have a large amount of pictures being processed and my slightly underpowered NAS will need weeks to complete it.

Now, my app upgraded to v106 hence I cannot login anymore and I must upgrade the server from v103 to v106 (with the docker compose file changes mentioned in the release notes, removing one container).

Question: will my jobs restart from scratch if I do this?

**Comment by bo0tzz** [comment, marked as answer]
No. Worst case is that the currently queued jobs get wiped out, but you can just readd those with the "missing" buttons in the jobs panel.

**Comment by Qhilm** [comment]
hmmm, well that didn't go well to be honest.

The outstanding "facial recognition" job went from 50k down to 9k, so something got lost for sure.

**Reply by bo0tzz** [reply]
That's just the queued jobs like I mentioned, which you can re-queue.

**Reply by Qhilm** [reply]
How do I requeue them without clearing everything and restarting processing for all photos? I thought I can only click "clear", which will simply reprocess all pictures in essence.

**Reply by bo0tzz** [reply]
The jobs in the queue are just tasks to do some calculation and store the result (recognised face/s). If you clear that you can then add new jobs back again either for all assets (reprocess everything like you say) or for those that don't have any results yet, which is what you want.

**Reply by Qhilm** [reply]
Ok so if some jobs disappeared I need to restart the whole process – that's unfortunately what I wanted to avoid, this takes weeks for me...

**Reply by bo0tzz** [reply]
I'm trying to tell you that you _don't_ need to restart the whole process. If you click "Missing" it'll pick right back up where it was.

**Reply by Qhilm** [reply]
aaaah, I got it now. There is no "missing" button for any job type where there is still queued jobs, only "pause" and "clear". I need to let the queue go down to 0 and then I will be able to click on "missing".

Sorry, that's the part that confused me.
